src/toffy/bin_extraction.py

```diff
@@ -99,12 +99,14 @@ def incomplete_fov_check(
     run_file_path = os.path.join(bin_file_dir, run_name + ".json")
     run_metadata = read_json_file(run_file_path, encoding="utf-8")
 
-    # get fov and channel info
-    fovs = io_utils.list_folders(extraction_dir, "fov")
+    # TODO: list_folders does not handle cases such as "fov0" correctly
+    # need to add a fix in to alpineer to deal with this
+    fovs = [f for f in os.listdir(extraction_dir) if "fov" in f]
+
+    # NOTE: the specifically-named "Au" channel is no longer be assumed to be present
+    # simply check the first "num_channels" channels found
     channels = io_utils.list_files(os.path.join(extraction_dir, fovs[0]), ".tiff")
     channels_subset = channels[:num_channels]
-    if "Au.tiff" not in channels_subset:
-        channels_subset = channels_subset[:-1] + ["Au.tiff"]
```

Comment:
Noah specifically noted [here](https://github.com/angelolab/toffy/pull/416/files#diff-e6488e9227107ba0a131998f74f9381c2f2a31047d4ad2d7fbc03398a88350a8) that Au should always be a default channel when checking for incomplete fovs. So if some panels don't have gold, then we need this if statement to change instead of removing it completely.
```if "Au.tiff" in channels  and not in channels_subset:```

Reply:
@camisowers in line with adding an explicit argument to set `intensity_channels` to the callback creation function, what about another explicit argument defining a custom name for the gold channel? By default, it will be set to `"Au"`, and even if the user leaves it as `None`, it'll still assume an `"Au.tiff"` is found.

In cases where some channels fail to load, instead of erroring out completely, check if the error is specifically caused by a failure to load in gold, and if so, throw a warning that the specific gold channel could not be located and to ask the user to double-check that this is indeed correct.